include/gatekeeper_net.h

```diff
@@ -497,6 +500,26 @@ hw_filter_eth_available(const struct gatekeeper_if *iface)
 	return iface->hw_filter_eth && iface->rss;
 }
 
+/*
+ * Compute the IPv4 checksum, either in hardware or software, depending
+ * on the capabilities of the NIC and the configuration.
+ *
+ * The IP header checksum field must be set to 0 before calling this function.
+ * Additionally, the pkt->l2_len and pkt->l3_len must be set before calling
+ * this function, although this is strictly only needed for hardware checksums.
```

Comment:
This is not a requirement of `set_ipv4_checksum()`: `The IP header checksum field must be set to 0 before calling this function.`; because `set_ipv4_checksum()` already does it.